SUPPORTED_APIS.md

```diff
@@ -2,31 +2,122 @@
 
 > ⚠️ **WORK IN PROGRESS**: This list is non-comprehensive and being updated
 
-# Key
+## Supported APIs Table
+
+| Namespace | API | Status | Description |
+| --- | --- | --- | --- |
+| [`CONFIG`](#config-namespace) | [`config_getShowCalls`](#config_getshowcalls) | SUPPORTED | Gets the current value of `show_calls` that's originally set with `--show-calls` option |
+| [`CONFIG`](#config-namespace) | [`config_setShowCalls`](#config_setshowcalls) | SUPPORTED | Updates `show_calls` to print more detailed call traces |
+| [`CONFIG`](#config-namespace) | [`config_setResolveHashes`](#config_setresolvehashes) | SUPPORTED | Updates `resolve-hashes` to call OpenChain for human-readable ABI names in call traces |
+| [`NETWORK`](#network-namespace) | [`net_version`](#net_version) | SUPPORTED | Returns the current network id |
+| [`NETWORK`](#network-namespace) | [`net_peerCount`](#net_peercount) |SUPPORTED | Returns the number of peers currently connected to the client |
+| [`NETWORK`](#network-namespace) | [`net_listening`](#net_listening) | SUPPORTED | Returns `true` if the client is actively listening for network connections |
+| `DEBUG` | `debug_storageRangeAt` | NOT IMPLEMENTED | Attempts to replay the transaction as it was executed on the network and return storage data |
+| `DEBUG` | `debug_traceTransaction` | PARTIALLY | Attempt to run the transaction in the exact same manner as it was executed on the network |
```

Comment:
I believe we should save this API endpoint has a status of `NOT IMPLEMENTED`. We do have the ability to to re-run transactions on a fork of mainnet/testnet, but this is not available via an API